analytics/analytics-ui/src/app/services/analytic-filter.pipe.ts

```diff
@@ -0,0 +1,18 @@
+import { Pipe, PipeTransform } from '@angular/core';
```

Comment:
Copyright header is needed for this method. Perhaps you could add something which automatically checks this.